drivers/sensor/sensor_shell.c

```diff
@@ -125,6 +125,35 @@ static const char *sensor_attribute_name[SENSOR_ATTR_COMMON_COUNT] = {
 	[SENSOR_ATTR_BATCH_DURATION] = "batch_dur",
 };
 
+enum sample_stats_state {
+	SAMPLE_STATS_STATE_UNINITIALIZED = 0,
+	SAMPLE_STATS_STATE_ENABLED,
+	SAMPLE_STATS_STATE_DISABLED,
+};
+
+#define MAX_NUM_TRIGGER_SENSOR_DEVICES 4
```

Comment:
Is this still needed? looks like it was converted to a Kconfig

Reply:
It isn't needed. It was from an earlier draft before I converted to Kconfig. I'll delete the line.